### PR TITLE
[Feat] Badge component

### PR DESCRIPTION
**Pull request by rushikesh3299**
Badge component for image & icon
https://deploy-preview-3--saraniui.netlify.app/components/badge/badge

### CODE REVIEW

**Review by Bharatrathod-81 — APPROVED** [review]
seems good

**Review by paarthbajaj — APPROVED** [review]
Well-written code. You can merge.

File: Components/Badge/badge.css [hunk 1]
```diff
@@ -1,2 +1,69 @@
...
+
+.badge-busy .badge{
+    background: var(--color-error);
+}
+
+.badge-icon-s .badge{
+    font-size: .7rem;
+    width: 1rem;
+    height: 1rem;
+    right:-0.5rem;
```
Comment (dikshadhote) [inline Components/Badge/badge.css:38]: use px for values less than 10 px as discussed in CSS session

File: Components/Badge/badge.css [hunk 2]
```diff
@@ -1,2 +1,69 @@
...
+    height: 1.4rem;
+    right:-0.7rem;
+    top:-0.7rem;
+}
+
+.badge-img-s .badge{
+    width: 1rem;
+    height: 1rem;
+    right:.2rem;
+    bottom:.2rem;
```
Comment (dikshadhote) [inline Components/Badge/badge.css:54]: Use px as mentioned above

File: Components/Badge/badge.css [hunk 3]
```diff
@@ -1,2 +1,69 @@
...
+}
+
+.badge-img-m .badge{
+    width: 1.4rem;
+    height: 1.4rem;
+    right:.35rem;
+    bottom:.35rem;
```
Comment (dikshadhote) [inline Components/Badge/badge.css:61]: same as mentioned above

File: Components/Badge/badge.html [hunk 4]
```diff
@@ -14,18 +14,69 @@
...
             <div class="nav-right">
                 <a href="#"><i class="fab fa-github"></i></a>
                 <a href="#"><i class="fab fa-twitter"></i></a>
                 <a href="#"><i class="fab fa-linkedin-in"></i></a>
             </div>
         </div>
-        <div class="side-bar">
-            
+        <div class="side-bar pb40">
+            <a href="/Components/Avatar/avatar.html" class="a--reset p20">Avatar</a>
```
Comment (dikshadhote) [inline Components/Badge/badge.html:26]: `class="a--reset` doesn't give clear indication of what this class is doing, will suggest you to use proper naming convention

**Review by dikshadhote — CHANGES_REQUESTED** [review]
Overall the good looks neat and clean and have proper naming conventions.
Will suggest you to add more description in PR like how many types of badges are there?
Really liked the hamburger for sidebar.
Please make the suggested changes as good practice.

**Pull request merged**